Source/Initialization/WarpXInitData.cpp

```diff
@@ -246,7 +246,8 @@ WarpX::InitNCICorrector ()
 void
 WarpX::InitFilter (){
     if (WarpX::use_filter){
-        WarpX::bilinear_filter.npass_each_dir = WarpX::filter_npass_each_dir;
+        for (int i = 0; i < AMREX_SPACEDIM; ++i)
```

Comment:
I added a new member to `amrex::IntVect` so you can do:
```suggestion
    WarpX::bilinear_filter.npass_each_dir = WarpX::filter_npass_each_dir.toArray<unsigned int>();
```
https://github.com/AMReX-Codes/amrex/pull/1768